--- libtool	2020-07-07 21:11:47.000000000 +0900
+++ libtool2	2020-07-07 21:12:12.000000000 +0900
@@ -10721,7 +10721,7 @@
 	# But is supposedly fixed on 10.4 or later (yay!).
 	if test CXX = "$tagname"; then
 	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
-	    10.[0123])
+	    10.[0123][,.])
 	      func_append compile_command " $wl-bind_at_load"
 	      func_append finalize_command " $wl-bind_at_load"
 	    ;;
