diff -rupN spatialite/src/gaiaaux/gg_utf8.c spatialite-ios/src/gaiaaux/gg_utf8.c
--- spatialite/src/gaiaaux/gg_utf8.c	2015-07-01 15:59:16.000000000 +0900
+++ spatialite-ios/src/gaiaaux/gg_utf8.c	2019-03-15 15:37:23.000000000 +0900
@@ -57,43 +57,15 @@ the terms of any one of the MPL, the GPL
 
 #if OMIT_ICONV == 0		/* ICONV is absolutely required */
 
-#if defined(__MINGW32__) || defined(_WIN32)
-#define LIBICONV_STATIC
-#include <iconv.h>
-#define LIBCHARSET_STATIC
-#ifdef _MSC_VER
-/* <localcharset.h> isn't supported on OSGeo4W */
-/* applying a tricky workaround to fix this issue */
-extern const char *locale_charset (void);
-#else /* sane Windows - not OSGeo4W */
-#include <localcharset.h>
-#endif /* end localcharset */
-#else /* not MINGW32 - WIN32 */
-#if defined(__APPLE__) || defined(__ANDROID__)
-#include <iconv.h>
-#include <localcharset.h>
-#else /* neither Mac OsX nor Android */
 #include <iconv.h>
 #include <langinfo.h>
-#endif
-#endif
-
 #include <spatialite/sqlite.h>
 #include <spatialite/gaiaaux.h>
 
 GAIAAUX_DECLARE const char *
 gaiaGetLocaleCharset ()
 {
-/* identifies the locale charset */
-#if defined(__MINGW32__) || defined(_WIN32)
-    return locale_charset ();
-#else /* not MINGW32 - WIN32 */
-#if defined(__APPLE__) || defined(__ANDROID__)
-    return locale_charset ();
-#else /* neither Mac OsX nor Android */
     return nl_langinfo (CODESET);
-#endif
-#endif
 }
 
 GAIAAUX_DECLARE int
diff -rupN spatialite/src/gaiageo/gg_shape.c spatialite-ios/src/gaiageo/gg_shape.c
--- spatialite/src/gaiageo/gg_shape.c	2015-07-01 15:59:16.000000000 +0900
+++ spatialite-ios/src/gaiageo/gg_shape.c	2019-03-15 15:38:35.000000000 +0900
@@ -59,29 +59,9 @@ the terms of any one of the MPL, the GPL
 
 #if OMIT_ICONV == 0		/* if ICONV is disabled no SHP support is available */
 
-#if defined(__MINGW32__) || defined(_WIN32)
-#define LIBICONV_STATIC
-#include <iconv.h>
-#define LIBCHARSET_STATIC
-#ifdef _MSC_VER
-/* <localcharset.h> isn't supported on OSGeo4W */
-/* applying a tricky workaround to fix this issue */
-extern const char *locale_charset (void);
-#else /* sane Windows - not OSGeo4W */
-#include <localcharset.h>
-#endif /* end localcharset */
-#else /* not MINGW32 */
-#if defined(__APPLE__) || defined(__ANDROID__)
-#include <iconv.h>
-#include <localcharset.h>
-#else /* neither Mac OsX nor Android */
 #include <iconv.h>
 #include <langinfo.h>
-#endif
-#endif
-
 #include <spatialite/sqlite.h>
-
 #include <spatialite/gaiageo.h>
 #include <spatialite/debug.h>
 
